api/v1alpha1/db_source.go

```diff
@@ -0,0 +1,16 @@
+package v1alpha1
+
+import (
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+type DBSource struct {
+	Database     `json:",inline" protobuf:"bytes,1,opt,name=database"`
+	Table        string `json:"table,omitempty" protobuf:"bytes,2,opt,name=table"`
+	OffsetColumn string `json:"offsetColumn,omitempty" protobuf:"bytes,3,opt,name=offsetColumn"`
+	// +kubebuilder:default="1s"
+	PollInterval metav1.Duration `json:"pollInterval,omitempty" protobuf:"bytes,4,opt,name=pollInterval"`
+	// +kubebuilder:default="5s"
+	CommitInterval metav1.Duration `json:"commitInterval,omitempty" protobuf:"bytes,5,opt,name=commitInterval"`
+	InitSchema     bool            `json:"initSchema,omitempty" protobuf:"bytes,6,opt,name=initSchema"`
```

Comment:
what is this for? maybe we just init always on leader start up?

Reply:
think about the case you are now allowed to have the permission to create tables and your admin helped you set up.